InnerEye/ML/deep_learning_config.py

```diff
@@ -222,6 +222,9 @@ class WorkflowParams(param.Parameterized):
     local_weights_path: List[Path] = param.List(default=[], class_=Path,
                                                 doc="A list of checkpoints paths to use for inference, "
                                                     "when the job is running outside Azure.")
+    experiment_name: str = param.String(default="",
+                                        doc="If provided, overrides the default AzureML experiment name "
+                                            "based on the current Git branch.")
```

Comment:
In the documentation, can you say how that interacts with the other experiment_name? (in the AzureConfig class) Also mention that in AzureConfig.experiment_name